java/src/main/java/vw/responses/ActionScores.java

```diff
@@ -0,0 +1,36 @@
+package vw.responses;
+
+import java.util.Arrays;
+
+/**
+ * Created by jmorra on 8/12/16.
+ */
+public class ActionScores {
+
+    private final ActionScore[] actionScores;
```

Comment:
Is it weird that you have a mutable "*collection *" that is private but then you return it directly in the getter method?  Should this be a `java.util.List` that is backed by `Collections.unmodifiableList`?  Maybe it will just suffice to make a javadoc comment that says that the backing array is returned directly and let the users make their own mistakes if they so choose.